installer/volcano-development-arm64.yaml

```diff
@@ -271,6 +274,8 @@ spec:
           imagePullPolicy: IfNotPresent
           command: ["./gen-admission-secret.sh", "--service", "volcano-admission-service", "--namespace",
                     "volcano-system", "--secret", "volcano-admission-secret"]
+      nodeSelector:
```

Comment:
Well, The introduction of `nodeSelector` requires that nodes in cluster must mark labels, which is not so flexible for common scenarios.

Reply:
AFAIK, this label will be added to node by kubelet when it start :)

Reply:
> AFAIK, this label will be added to node by kubelet when it start :)

If that it can be acceptable. We need a new yaml for spark.

Reply:
The new Spark (3.1) has a flag to cross build for arm & amd64 which is probably the better approach long term.

Reply:
Oh yes and this label is added automatically you don't have to tag them manually.

Reply:
> Oh yes and this label is added automatically you don't have to tag them manually.

Great. Let me submit a new yaml for spark.

Reply:
Awesome, if your going to rebuild the spark containers `-X` is the flag to cross build with buildx using the docker-image-tool script in Spark :)

Reply:
> Awesome, if your going to rebuild the spark containers `-X` is the flag to cross build with buildx using the docker-image-tool script in Spark :)

OK, Thanks!